test/envtest/suite_test.go

```diff
@@ -229,6 +440,27 @@ func makeDevicePlugin(name, image, initimage string) client.Object {
 	return obj
 }
 
+func makeDP(name string, spec interface{}) client.Object {
```

Comment:
@ozhuraki I made this to ask if it can replace with the function makeDevicePlugin(). I also need a similar function like makeDevicePlugin(), but since I need to test other things in spec than image and initimage, it would be good if we can just put spec as a parameter. Wdyt?